code/game/objects/items/weapons/grenades/plastic.dm

```diff
@@ -89,7 +89,7 @@
 /obj/item/weapon/grenade/plastic/suicide_act(mob/user)
 	message_admins("[key_name_admin(user)](<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[user]'>FLW</A>) suicided with [src] at ([user.x],[user.y],[user.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)",0,1)
 	message_admins("[key_name(user)] suicided with [src] at ([user.x],[user.y],[user.z])")
-	user.visible_message("<span class='suicide'>[user] activates [src] and holds it above [user.p_their()] head! It looks like [user.p_theyre()] going out with a bang!</span>")
+	user.visible_message("<span class='suicide'>[user] activates the [src] and holds it above [user.p_their()] head! It looks like [user.p_theyre()] going out with a bang!</span>")
```

Comment:
Just tried using that. \improper only works at the beginning of a sentence, and putting it here prevents it from compiling. The only way it would work is if I had a temporary variable with the name, and I don't see why I could just keep it as-is. If a maintainer _really_ wants it, I'll do it.